src/components/tooltip/tooltip.js

```diff
@@ -86,6 +93,10 @@ function MdTooltipDirective($timeout, $window, $$rAF, $document, $mdUtil, $mdThe
       }
     }
 
+    function setTransformOrigin (direction) {
```

Comment:
What is the intent of this function? 
Why is it not just contained (as a closure) in `configureWatchers( ... ) `?